.github/workflows/deploy-staging.yaml

```diff
@@ -48,6 +48,18 @@ jobs:
         DOCKER_PRUNE: '1'
         KURL_UTIL_IMAGE: replicated/kurl-util:alpha
         KURL_BIN_UTILS_FILE: kurl-bin-utils-latest.tar.gz
+  
+  build-addons:
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v2
+
+    - run: make generate-addons
+      env:
+        S3_BUCKET: kurl-sh
+        AWS_ACCESS_KEY_ID: ${{ secrets.AWS_STAGING_ACCESS_KEY_ID }}
+        AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_STAGING_SECRET_ACCESS_KEY }}
```

Comment:
These should be `AWS_PROD_ACCESS_KEY_ID` and `AWS_PROD_SECRET_ACCESS_KEY`. The `staging/` folder is in the prod bucket.